api/src/pcapi/routes/native/v1/serialization/account.py

```diff
@@ -159,6 +155,15 @@ def from_model(
         )
         return subscription_message
 
+    @classmethod
+    def beneficiary_maintenance_message(cls) -> "SubscriptionMessage":
+        return cls(
+            userMessage="La vérification d'identité est momentanément indisponible. L'équipe du pass Culture met tout en oeuvre pour la rétablir au plus vite.",
+            callToAction=None,
+            popOverIcon=subscription_models.PopOverIcon.CLOCK,
+            updatedAt=datetime.datetime.now(),
```

Comment:
```suggestion
            updatedAt=datetime.datetime.utcnow(),
```

Pas d'impact sur la prod qui est en UTC, mais ça l'est en local ou dans les tests (hors CI) lancés sur un système dont le fuseau horaire n'est probablement pas UTC.